tests/types/ember-concurrency-test.ts

```diff
@@ -0,0 +1,1883 @@
+import ClassicComponent from '@ember/component';
+import EmberObject, { get } from '@ember/object';
+import Evented from '@ember/object/evented';
+import GlimmerComponent from '@glimmer/component';
+import {
+  Task,
+  TaskFunction,
+  TaskFunctionArgs,
+  TaskFunctionReturnType,
+  TaskGenerator,
+  TaskGroup,
+  TaskGroupProperty,
+  TaskInstance,
+  TaskProperty,
+  all,
+  allSettled,
+  didCancel,
+  forever,
+  hash,
+  race,
+  rawTimeout,
+  task,
+  taskGroup,
+  timeout,
+  waitForEvent,
+  waitForProperty,
+  waitForQueue
+} from 'ember-concurrency';
+import { restartableTask } from 'ember-concurrency-decorators';
+import { expectTypeOf as expect } from 'expect-type';
```

Comment:
I did the search/replace but left the import name. I think for the most part, `expect(...).toBeString()` etc actually reads fine, and I think the main reason to emphasize on the `TypeOf` part is when used along side jest etc where `expect` is already taken as a name, since we aren't here, I think the shorter name is nice and makes it easier to read. (And I don't want to have to reindent a bunch of lines in case if width changes too much.)